dp/_visualizer.py

```diff
@@ -306,6 +306,7 @@ def _create_figure(self, arr, colorscale_name="Sunset"):
             layout={
                 "title": arr.array_name,
                 "title_x": 0.5,
+                "height": max(100 * h, 300),
```

Comment:
Not sure I'm a fan of the borders, but I'm not sure if that's just because I'm used to the old version.